phpstan.neon

```diff
@@ -1,7 +1,9 @@
 parameters:
-    checkMissingIterableValueType: false
-    checkGenericClassInNonGenericObjectType: false
+    ignoreErrors:
+        - identifier: missingType.iterableValue
+        - identifier: missingType.generics
     level: max
     paths:
         - src
         - test
+    treatPhpDocTypesAsCertain: false
```

Comment:
Any particular reason for this parameter? In other repos we did not have to add this

Reply:
It was to cover off a couple of errors. I've reduced that number to one, but I think we're a bit stuck because of some dodgy typing in Laminas. See `src/CpmsCommon/View/JsonExceptionStrategy.php` line 112. If you click through to the MVCEvent, then it only returns `ApplicationInterface` for `getApplication`, rather than `ApplicationInterface|null`.

Reply:
I've removed the config and will ignore the above for now, unless anyone has any suggestions?